rct229/rulesets/ashrae9012019/section19/section19rule11.py

```diff
@@ -0,0 +1,67 @@
+from rct229.rule_engine.partial_rule_definition import PartialRuleDefinition
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.rulesets.ashrae9012019.ruleset_functions.baseline_system_type_compare import (
+    baseline_system_type_compare,
+)
+from rct229.rulesets.ashrae9012019.ruleset_functions.baseline_systems.baseline_system_util import (
+    HVAC_SYS,
+)
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_baseline_system_types import (
+    get_baseline_system_types,
+)
+
+APPLICABLE_SYS_TYPES = [
+    HVAC_SYS.SYS_11_1,
+]
+
+
+class Section19Rule11((RuleDefinitionListIndexedBase)):
+    """Rule 11 of ASHRAE 90.1-2019 Appendix G Section 19 (HVAC - General)"""
+
+    def __init__(self):
+        super(Section19Rule11, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section19Rule11.HVACRule(),
+            index_rmr="baseline",
+            id="19-11",
+            description="For systems that serve computer rooms, if the baseline system is HVAC System 11, it shall include an integrated fluid economizer meeting the requirements of Section 6.5.1.2 in the baseline building design.",
+            ruleset_section_title="HVAC - General",
+            standard_section="Section G3.1.2.6.1",
+            is_primary_rule=False,
+            rmr_context="ruleset_model_descriptions/0",
+            list_path="$.buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+        )
+
```

Comment:
You can also create an `is_applicable` here to check if there is at least one system type 11.
I am OK both way.